src/ngx_http_lua_util.h

```diff
@@ -500,7 +506,7 @@ ngx_http_lua_check_unsafe_string(ngx_http_request_t *r, u_char *str, size_t len,
                      /* %00-%1F, %7F */
```

Comment:
You forgot the update this comment to reflect the change below?

I believe a general "unsafe string" makes no sense (as I mentioned in a review comment yesterday). Headers, URIs, URI arguments may have different "unsafe" byte ranges anyway. We'd better be more explicit about our intention.

BTW, do we really need this function now?